### PR TITLE
Update log4net docs

### PR DESCRIPTION
**Pull request by chrisdunelm**
Add ASP.NET example

Update configuration options

### CODE REVIEW

File: apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/readme.md [hunk 1]
```diff
@@ -40,17 +40,25 @@ log.Info("An exciting log entry!");
...
+
+### projectId
 
 ``` xml
 <projectId value="<your project ID>"/>
 ```
-Must be present. The project ID given must be the ID of a project that already exists on [Google Cloud Platform](https://cloud.google.com/).
+Optional when running on Google App Engine (GAE) or Google Compute Engine (GCE), must be present on other platforms. The project ID given must be the ID of a project that already exists on [Google Cloud Platform](https://cloud.google.com/).
+
+On GCE and GAE the projectId will be auto-detected from the platform.
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/readme.md]: Nit: projectId => project ID

Comment (chrisdunelm) [inline apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/readme.md]: Done

**Comment by chrisdunelm** [comment]
I think the xml problem is fixed.
But now both appveyor and travis appear to have frozen :(

**Comment by jskeet** [comment]
Nope, still "Tag name log4net_aspnet_template is not found" - do the docs build locally for you?

**Comment by chrisdunelm** [comment]
Meant to say a few days ago...
PTAL :)

**Comment by jskeet** [comment]
LGTM. I'd forgotten the whole resource syntax :)

**Pull request merged**